### ISSUE TITLE
Webpack 4 - DeprecationWarning: Chunk.forEachModule

### ISSUE THREAD

**Issue by mtraynham** [xz64/license-webpack-plugin#26]
Webpack 4 is complaining of a deprecation warning:

```
DeprecationWarning: Chunk.forEachModule: Use for(const module of chunk.modulesIterable) instead
```

I believe it relates to [this particular line](https://github.com/xz64/license-webpack-plugin/blob/master/src/LicenseWebpackPlugin.ts#L128).  Hopefully I can get around to a PR for this.

**Comment by xz64** [comment]
Fixed in v1.2.0

**Comment by xz64** [comment]
Initial fix for this was causing issues. (It is reverted now) I need to look into this more. I suspect it has something to do with typescript transpilation.

**Comment by xz64** [comment]
Fixed in v1.2.3.

Needed to turn on `"downlevelIteration": true` in tsconfig.json
